src/main/java/io/stargate/sgv2/jsonapi/exception/mappers/ThrowableToErrorMapper.java

```diff
@@ -180,6 +187,19 @@ private static CommandResult.Error handleAllNodesFailedException(
     return handleUnrecognizedException(throwable, message);
   }
 
+  private static CommandResult.Error handleJsonProcessingException(
+      JacksonException e, String message) {
+    // Low-level parsing problem?
+    if (e instanceof JsonParseException) {
+      return ErrorCode.INVALID_REQUEST_NOT_JSON
+          .toApiException("underlying problem: (%s) %s", e.getClass().getName(), message)
+          .getCommandResultError(Response.Status.OK);
+    }
+
+    // Will need to add more handling but start with above
+    return handleUnrecognizedException(e, message);
```

Comment:
Agreed on the comment above. Since, we know this is one of `JacksonException`, we can keep adding more known jackson errors.

Reply:
Exactly. I plan to follow up on this asap.